src/Controller/CRUDController.php

```diff
@@ -1337,11 +1335,13 @@ protected function isPreviewDeclined()
     }
 
     /**
-     * Gets ACL users.
+     * Gets ACL users, or selected user if $id is provided
+     *
+     * @param string $id|null   User id
```

Comment:
Sure this is always an int?

Reply:
String is more appropriate for ODM/ORM. Tested with mongodb only here.

Reply:
If this could be of any type, please use `mixed` as type hint

Reply:
You could also change the parameter name to $userId and remove the description